packages/build/src/error/info.js

```diff
@@ -11,10 +19,11 @@ const addErrorInfo = function (error, info) {
 
 const getErrorInfo = function (error) {
```

Comment:
[sand] Would it make sense to add a comment to this function explaining its signature, especially the return value? It may not be immediately obvious what the returned array contains.

Reply:
:+1: 
Done in 963044a680067984dc3e13a438a60523a318290b